commitrmnocb/types.go

```diff
@@ -0,0 +1,255 @@
+package commitrmnocb
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+
+	"github.com/smartcontractkit/chainlink-ccip/plugintypes"
+
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+)
+
+type Query struct {
+	RmnOnRampMaxSeqNums []plugintypes.SeqNumChain
+	MerkleRoots         []cciptypes.MerkleRootChain
+}
+
+func (q Query) Encode() ([]byte, error) {
+	return json.Marshal(q)
```

Comment:
Is it important that the encoding of the Query be deterministic? If so, lets add some `sort.Slice`s here to make sure that the slices in the struct are sorted (perhaps according to chain selector, like the other sorts)

Reply:
I'll add a TODO to ensure we do this when Query has fields